### PR TITLE
Add documentation for deleting an app

### PR DESCRIPTION
**Pull request by wmora**
@jpjoyal @alavers @mspensieri @dannytranlx @hebime

### CODE REVIEW

**Comment by wmora** [comment]
[Documentation in internal docs](https://trello.com/c/KZzCC7hB/4793-documentation-in-internal-docs)

File: source/includes/rest/_inside.md [hunk 1]
```diff
@@ -159,6 +159,30 @@ curl https://api.smooch.io/v1/apps/55c8d9758590aa1900b9b9f6 \
...
+
+> Response:
+
+```
+204 NO CONTENT
+```
+
+<api>`DELETE /v1/apps/{appId}`</api>
+
+Removes the specified app.
```
Comment (jpjoyal) [inline source/includes/rest/_inside.md]: Might want to add that it remove each enable integration at the same time

**Comment by wmora** [comment]
@jpjoyal done!

**Comment by wmora** [comment]
Will change the response code to 200

**Pull request merged**